readthedocs/worker.py

```diff
@@ -17,3 +18,14 @@ def create_application():
 
 
 app = create_application()  # pylint: disable=invalid-name
+
+
+@app.on_after_finalize.connect
```

Comment:
@agjohnson Do you have any conern about using this approach to register beat task?

Reply:
As far as I know, we are not using this approach but registering the beat task from a setting in the operations repository (private).

Example:

```
    CELERYBEAT_SCHEDULE = {
        'quarter-finish-inactive-builds': {
            'task': 'readthedocs.projects.tasks.finish_inactive_builds',
            'schedule': crontab(minute='*/15'),
            'options': {'queue': 'web'},
        },
    }
```

So, I suppose that we prefer that way as a general rule.

Reply:
I think its better to have this in our code rather than the private operation repository. So other users of Read the Docs platform can be benifical from it. What do you think @humitos ?

Reply:
This should be managed via settings in this repository, not in code. We have some we add in ops, but that's different.